src/snowflake/snowpark/_internal/data_source/utils.py

```diff
@@ -53,6 +53,12 @@ class DBMS_TYPE(Enum):
     "sqlite3": SqliteDriver,
 }
 
+UDTF_PACKAGE_MAP = {
+    DBMS_TYPE.ORACLE_DB: ["oracledb", "snowflake-snowpark-python"],
+    DBMS_TYPE.SQLITE_DB: ["sqlite3", "snowflake-snowpark-python"],
```

Comment:
`sqlite3` is a python built-in module

```suggestion
    DBMS_TYPE.SQLITE_DB: ["snowflake-snowpark-python"],
```